AmplifyPlugins/Auth/AWSAuthPlugin/Operations/AWSAuthSignUpOperation.swift

```diff
@@ -47,6 +47,7 @@ AuthSignUpOperation {
             case .failure(let signUpError):
                 self.dispatch(signUpError)
             }
+            self.finish()
```

Comment:
Change this to a `defer` block at the top of the method, then you can remove it from the `isCancelled` block as well.

Reply:
Added